test/piggyback-tests.js

```diff
@@ -112,8 +112,8 @@ test2('ringpop piggybacking decays', getClusterSizes(3), 40000, prepareCluster(f
         dsl.waitForEmptyPing(t, tc), // problem is that if decay is not working you might never get to this point
 
         // send information to be piggy backed
-        dsl.changeStatus(t, tc, 0, 1, 'suspect'),
-        dsl.waitForPingResponse(t, tc, 0),
+        dsl.changeStatus(t, tc, 1, 2, 'suspect'),
```

Comment:
Need to send ping from different node because the SUT filters out changes that originated from the node. The suspect change now originates from node 1, and node 0 sends the pings to drain the SUT.